cyral/data_source_cyral_integration_logs.go

```diff
@@ -0,0 +1,95 @@
+package cyral
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/cyralinc/terraform-provider-cyral/client"
+	"github.com/google/uuid"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+type ListIntegrationLogsRequest struct {
+	IntegrationType string `json:"type"`
+}
+
+type ListIntegrationLogsResponse struct {
+	Integrations []IntegrationLogConfig `json:"integrations"`
+}
+
+func (resp *ListIntegrationLogsResponse) WriteToSchema(d *schema.ResourceData) error {
+	var integrationList []interface{}
+	for _, integration := range resp.Integrations {
+		// write in config scheme
+		configScheme, err := writeConfigScheme(&integration)
+		if err != nil {
+			return err
+		}
+
+		integrationList = append(integrationList, map[string]interface{}{
```

Comment:
Should be `integrationsList`. Missed the plural. Maybe using just `integrations` makes more sense as this is not an actual list data structure.

Reply:
Addr here: [e7a72a9](https://github.com/cyralinc/terraform-provider-cyral/pull/395/commits/e7a72a9c6616ff48fb3487ee132491a3900f7939)